packages/workbox-core/src/_private/dontWaitFor.ts

```diff
@@ -12,7 +12,7 @@ import '../_version.js';
  *
  * @private
  **/
-export function dontWaitFor(promise: Promise<any>) {
+export function dontWaitFor(promise: Promise<any>): void {
```

Comment:
I think the `tsc` dependency bump did something to make this required.

https://github.com/GoogleChrome/workbox/pull/2608/checks?check_run_id=991709793#step:6:131